easybuild/easyconfigs/h/Hyperspace/Hyperspace-2.2.5-Java-11.eb

```diff
@@ -0,0 +1,40 @@
+easyblock = "PackedBinary"
+
+name = 'Hyperspace'
+version = '2.2.5'
+
+versionsuffix = '-Java-%(javaver)s'
+
+homepage = 'https://github.com/Actelion/openchemlib-hyperspace'
+description = """Cheminformatics tools, workflows and pipelines 
+for substructure search, virtual screening and data analysis."""
+
+toolchain = SYSTEM
+
+sources = [{
+    'filename': '%(name)s-%(version)s.tar.gz',
```

Comment:
We probably have to use the `bz` variant here to ensure the reproducibility of the tarball hash